x-pack/plugins/apm/public/components/app/service_inventory/index.tsx

```diff
@@ -265,6 +266,34 @@ export function ServiceInventory() {
     [tiebreakerField]
   );
 
+  const setApplicationContext =
+    useApmPluginContext().observabilityAIAssistant.service
+      .setApplicationContext;
+
+  useEffect(() => {
+    if (mainStatisticsStatus === FETCH_STATUS.FAILURE) {
+      return setApplicationContext({
+        description: 'The services have failed to load',
```

Comment:
Does it make sense to add `i18n` here and on the other descriptions below?

Reply:
No, everything we send to the LLM is in English